tools/osbuild-image-test

```diff
@@ -27,6 +29,50 @@ RED = "\033[31m"
 YELLOW = "\033[33m"
 
 
+def filter_fn(imi):
+    """
+    Filter specific fields in the image-info that can't be compared
+    together.
+    """
+    def tmpfiles_d(imi):
+        to_remove = ["rpm-ostree-1-autovar.conf"]
```

Comment:
nitpick: It would be IMO much nicer to be able to pass this to the test case as an argument, instead of hardcoding it in the test itself. Nevertheless I do not have strong opinion on this and would find this acceptable.

Reply:
I prefer leaving this as it is. I don't want to add too much parameters. And if we need to extend this list, it's good to have it where it's executed.

Reply:
In the past, we ran into different issues on different distributions, Therefore applying the same rules on each distribution is not the best idea to do. But as I've said, I'm able to live with this version.

Reply:
IC, so, I think, we should store in the DB entries which files to ignore then.

Reply:
This is now stored in the DB, and only one distribution gets it (for now)